contrib/pkg/awsprivatelink/disable.go

```diff
@@ -16,9 +16,7 @@ import (
 	"github.com/spf13/cobra"
 )
 
-type disableOptions struct {
-	dynamicClient client.Client
-}
+type disableOptions struct{}
```

Comment:
Keeping an empty struct to adhere to the same paradigm as the other subcommands. Not sure if it's worth it though.

Reply:
I kind of thought `Complete`, `Validate`, and `Run` were part of some interface, but they don't appear to be. I can't decide which pains my OCD more, having unused cruft or deviating from an established pattern. I'll leave it up to you :)

[Later] If you compose the common CredsSecret and DynamicClient in, this point becomes moot :P